src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -57,6 +57,15 @@ function getUnmaskedContext(workInProgress : Fiber) : Object {
 }
 exports.getUnmaskedContext = getUnmaskedContext;
 
+function cacheContext(workInProgress : Fiber, unmaskedContext : Object, maskedContext : Object) {
+  const instance = workInProgress.stateNode;
+  if (instance) {
```

Comment:
Sure, we could do that. I didn't do it initially because it seemed unsafe. `Fiber` may not have an `instance` and I didn't wan to rely on external callers of this function having to know only to call this method when an `instance` existed.